app/views/provider_interface/organisation_settings/show.html.erb

```diff
@@ -3,18 +3,36 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l"><%= t('page_titles.provider.organisation_settings') %></h1>
-    <ul class="govuk-list govuk-list--spaced">
-      <% if current_provider_user.authorisation.can_manage_users_for_at_least_one_provider? %>
-        <li>
-          <%= govuk_link_to t('page_titles.provider.users'), provider_interface_provider_users_path %>
-        </li>
+    <% if FeatureFlag.active?(:account_and_org_settings_changes) %>
+      <% @providers.each do |provider| %>
+        <% if @providers.count > 1 %>
+          <h2 class="govuk-heading-m"><%= provider.name %></h2>
+        <% end %>
+        <ul class="govuk-list govuk-list--spaced">
+          <li>
+            <%= govuk_link_to t('page_titles.provider.users'), provider_interface_provider_users_path %>
+          </li>
+          <% if ProviderRelationshipPermissions.all_relationships_for_providers([provider]).governs_open_course.any? %>
```

Comment:
Just to confirm, this doesn't need to do a permissions check because any user will be able to view existing permissions.